ui/pages/routes/routes.component.test.js

```diff
@@ -137,6 +144,111 @@ describe('Routes Component', () => {
       expect(getByTestId('account-menu-icon')).not.toBeDisabled();
     });
   });
+
+  describe('new network popup', () => {
+    const mockBtcAccount = createMockInternalAccount({
+      address: 'bc1q26a367uz34eg5mufwhlscwdcplu6frtgf00r7a',
+      name: 'Btc Account',
+      type: BtcAccountType.P2wpkh,
+      snapOptions: {
+        id: 'mock-btc-snap',
+        name: 'mock-btc-snap',
+        enabled: true,
+      },
+    });
+
+    const mockEvmAccount = createMockInternalAccount();
```

Comment:
We could re-use the accounts defined in `mock-accounts.ts`:

```suggestion
    const mockBtcAccount = MOCK_ACCOUNT_BIP122_P2WPKH;
    const mockEvmAccount = MOCK_ACCOUNT_EOA;
```